### PR TITLE
feat: Secondary One and Columnist Slice

### PR DESCRIPTION
**Pull request by jimhunty**
Secondary One and Columnist Slice: https://nidigitalsolutions.jira.com/browse/REPLAT-5144

screenshot 2019-02-19 at 15 03 12

### CODE REVIEW

**Comment by aronshamash** [comment]
Looks good 👍. Couple of small design things:

- looks like we need and extra `paddingBottom: spacing(1)` for the headline in the top tile
- looks like there is too much margin between the flags and byline

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 1]
```diff
@@ -13,8 +13,9 @@ const TileSummary = ({
     article: { flags, hasVideo, headline, label, section, shortHeadline }
   },
   byline,
-  headlineStyle,
   bylineStyle,
+  headlineStyle,
+  isOpinionByline = true,
```
Comment (nehasri89) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: this has changed to `bylineStyle` now in master

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 2]
```diff
@@ -26,7 +27,7 @@ const TileSummary = ({
   <ArticleSummary
     bylineProps={
       byline
-        ? { ast: byline, isOpinionByline: true, opinionStyle: bylineStyle }
+        ? { ast: byline, isOpinionByline, opinionStyle: bylineStyle }
```
Comment (nehasri89) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: `opinionStyle` has changed to `bylineStyle` now in master

File: packages/edition-slices/src/tiles/tile-h/index.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const TileH = ({ onPress, tile }) => (
       <TileSummary
         byline={tile.article.byline}
         headlineStyle={styles.headline}
+        isOpinionByline={false}
```
Comment (nehasri89) [inline packages/edition-slices/src/tiles/tile-h/index.js]: should not need this

**Comment by jimhunty** [comment]
@aronshamash My understanding is that these tweaks, which are part of `ArticleSummary` will come in a design review. I agree it doesn't look great.

**Comment by aronshamash** [comment]
Personally I think we should get as close as possible to the designs in the initial PR

**Comment by jimhunty** [comment]
@aronshamash This would have been the tile PR not this slice one. A tile that I didn't work on. I would suggest getting this in and going to design review after.

**Pull request merged**